### ISSUE TITLE
disabled options: navigate by keyboard

### ISSUE THREAD

**Issue by fnagel** [fnagel/jquery-ui#116]
Parent: https://github.com/fnagel/jquery-ui/issues/89

on the disable option demo, disable the option. tab into the corresponding selectmenu. try to navigate past the disabled option. Note that you cannot.

Note: this only occurs when menu is closed

**Comment by fnagel** [comment]
Possible solution: https://github.com/fnagel/jquery-ui/issues/104

**Comment by fnagel** [comment]
Seems to be fixed, see issue posted above.
